python/ray/serve/_private/constants.py

```diff
@@ -168,6 +168,9 @@
 # future. Use RAY_SERVE_LOG_ENCODING or 'LoggingConfig' to enable json format.
 RAY_SERVE_ENABLE_JSON_LOGGING = os.environ.get("RAY_SERVE_ENABLE_JSON_LOGGING") == "1"
 
+# Disable logging to the stdout and stderr. Also, redirect them to the serve's log files
+RAY_SERVE_DISABLE_STDOUT = os.environ.get("RAY_SERVE_DISABLE_STDOUT") == "1"
```

Comment:
Open for other naming suggestions. I kinda feel this is not descriptive enough, but also don't want to name it something like `RAY_SERVE_DISABLE_STDOUT_STDERR_AND_REDIRECT_TO_SERVE_LOGGER`😅

Reply:
Maybe `RAY_SERVE_LOG_TO_STDOUT` and set to `0` or `1`? It's not quite 100% correct but gets the point across

Reply:
or actually probably `RAY_SERVE_LOG_TO_STDERR` (to match the `extra` tag)